src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -2527,6 +2527,7 @@ elementFormDefault="qualified">
             <xs:complexContent>
                 <xs:extension base="msb:TaskType">
                     <xs:attribute name="Filtered" />
+                    <xs:attribute name="HadAnyDuplicates" type="msb:boolean" />
```

Comment:
Is there anything that needs to be done to make sure that the docs.microsoft.com documentation gets update to talk about the new output parameter?

Reply:
Unfortunately it's a manual process; we'll have to send a PR after this is merged/shipped.

Reply:
:) https://github.com/MicrosoftDocs/visualstudio-docs/pull/8832